adam/learner/subset.py

```diff
@@ -132,10 +132,10 @@ class AbstractSubsetLearnerNew(AbstractTemplateLearnerNew, ABC):
     _concept_to_hypotheses: Dict[Concept, ImmutableSet[PerceptionGraphTemplate]] = attrib(
         init=False, default=Factory(dict)
     )
-    _concept_to_surface_template: Dict[Concept, SurfaceTemplate] = attrib(
+    concept_to_surface_template: Dict[Concept, SurfaceTemplate] = attrib(
```

Comment:
Should these be public?

Reply:
maybe not for this case, but accessibility comes in handy for semantics